images/devtools-golang-v1beta1/context/Dockerfile

```diff
@@ -12,6 +12,7 @@ FROM docker.io/library/golang:1.21.1-bullseye@sha256:436969571fa091f02d34bf2b9bc
 SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 
 RUN \
+    go env -w GOPRIVATE=github.com/coopnorge/* && \
     go env > /usr/local/bin/goenv.sh && \
```

Comment:
I'm probably just missing it, but what uses this `goenv.sh` env script?

Reply:
this was just a test, turn I was looking the wrong direction